03_launch_mgmt_cluster.sh

```diff
@@ -273,6 +273,10 @@ function apply_bm_hosts() {
 function update_capm3_imports(){
   pushd "${CAPM3PATH}"
 
+  # Assign empty secret to BMO when TLS disable (BMO as part of CAPM3).
```

Comment:
since we have BMO as part of CAPM3 in both api versions for now, we do not need to specify it here.

Reply:
Done